lib/copy.js

```diff
@@ -0,0 +1,15 @@
+
+
+const fs = require('fs').promises;
+
+const copy = async(src, dst) => {
+  const file = await fs.readFile(src, { encoding: 'utf8' });
+  const retrunFile =  fs.writeFile(dst, file);
```

Comment:
Your test isn't passing because you need to await fs.writeFile and then return the original file...
const copy = async(src, dst) => {
  const file = await fs.readFile(src, { encoding: 'utf8' });
  await fs.writeFile(dst, file);
  return file;
};